forge/test/mlir/mnist/training/test_training.py

```diff
@@ -410,3 +410,80 @@ def test_loss_device():
             break
 
     print(f"Test (total) loss: {test_loss}")
+
+
+@pytest.mark.push
+def test_lora():
+    torch.manual_seed(0)
+
+    # Config
+    num_epochs = 3
+    batch_size = 1
```

Comment:
We could use a batch size larger than 1 here and no gradient accumulation to simulate a standard training scenario?